src/Sarif.Multitool.FunctionalTests/Rules/ValidationSkimmerTestsBase.cs

```diff
@@ -87,15 +88,31 @@ protected void Verify(string testFileName, bool disablePrereleaseCompatibilityTr
         // Verify that those detected result locations match the expected locations.
         private void Verify(Run run, ExpectedValidationResults expectedResults)
         {
-            string[] detectedResultLocations = run.Results.Select(r => r.Message.Arguments[0]).OrderBy(loc => loc).ToArray();
-            string[] expectedResultLocations = expectedResults.ResultLocationPointers.OrderBy(loc => loc).ToArray();
+            HashSet<string> actualResultLocations = new HashSet<string>(run.Results.Select(r => r.Message.Arguments[0]).OrderBy(loc => loc));
```

Comment:
>.OrderBy(loc => loc) [](start = 119, length = 20)

Since you're making a `HashSet`, there's no reason to order the list.

Reply:
Got it. FYI, i have updated this PR to merge files-array, will push an update in a moment.